jobs/config.json

```diff
@@ -310,14 +310,16 @@
   "ci-cri-containerd-e2e-gci-gce-ingress": {
     "args": [
       "--check-leaked-resources",
+      "--cluster=",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gci-gce-ingress.env",
```

Comment:
we are moving towards eliminating .env files... simply reference the three envs use `--env` flag here?

Reply:
Offline discussed with @krzyzacy, he seems fine with this now. :P

Reply:
Curious -- many jobs share the same env file. Is there a way for them to reuse the same set of envs without a file?

Reply:
Later on all the envs should be moved to prow config, and we can create presets for them instead.

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L195-L209 for example.

we want to get rid of config.json entirely.